### ISSUE TITLE
Printing problem non-standard \Rule

### ISSUE THREAD

**Issue by ozdaniel9** [mathjax/MathJax#1467]
Just today (11 May 2016) I noticed the non-standard command \Rule{}{}{} does not appear in print (PDF) any more, although it still does in browsers. The verticla line is invisible. This is the case in particular with Carol Burns page http://www.onemathematicalcat.org/MathJaxDocumentation/TeXSyntax.htm, using Chrome and Firefox.
Does MathJax 2.6 require adding something to fix this? 

Thanks.

**Comment by dpvc** [comment]
The rule is implemented via an `<mpadded>` element of the proper size with a background color.  Many browsers don't print background colors by default, but allow you to turn on background colors optionally.  You need to enable background colors to get them to print.  This has been the case all along, so is not new in version 2.6.

**Comment by ozdaniel9** [comment]
Dear Davide,

Thank you very much for your email. However, I was not talking about 
background colour at all,  there is a mixup here.

I was noting that all of a sudden the output of the tex command 
"\Rule{}{}{}" was invisible in a PDF obtained by printing the web page 
that included that mathjax command. I am not very knowledgeable about 
how the tex output is geenrated, this could be anything, could this be 
temporary? Please excuse my ignorance.

All the best, and thanks for working on mathjax, I appreciate it a lot, 
I have been using TeX/LaTeX for decades (as a university lecturer), and 
I have recently decided to go HTML/MathJax for most of what I write.

Daniel

**Comment by dpvc** [comment]
As I said in my earlier post, `\Rule` is implemented via a background color, so you *are* talking about background colors.  If you don't have printing of background colors enabled, the `\Rule` will not print.  That is true in version 2.6, but was also true all the way back to version 1.0.  If you were getting the rules to print before, perhaps your browser's defaults were set to include background color printing, and that has changed.  Did you switch or upgrade your browser recently?

**Comment by ozdaniel9** [comment]
Dear Davide,

Sincere apologies for not reading your first email properly.

I tried with all the different browsers and wrote a whole story about 
it, before finding out that there is a 'background graphics' setting in 
Chrome:Print, and checking that box does make the vertical lines 
(produced with \Rule) reappear. This solves my problem for the most 
part, I'll just have to write somewhere on my web pages that if printing 
the page people need to check that box too to make all math to print; or 
else produce PDFs myself and add a separate link.

I have googled 'force browser to print background graphics' (e.g. 
https://wiki.csswg.org/ideas/print-backgrounds) but did not find a way 
to do so.

Thanks again for taking the time to write back. I have learned something.

Cheers,

Daniel
